drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -151,6 +148,7 @@ def items_without_answers():
             for item in items:
                 del item['feedback']
                 del item['zone']
+                item['displayName'] = self.ugettext(item['displayName'])
```

Comment:
@itsjeyd can you please highlight the places where `ugettext` is not required anymore? I am going to update this PR with version as well.